src/main/java/frc/lib/team3061/drivetrain/Drivetrain.java

```diff
@@ -434,6 +447,16 @@ public void driveFacingAngle(
       xVelocity *= slowModeMultiplier;
       yVelocity *= slowModeMultiplier;
     }
+
+    if (Constants.DEMO_MODE) {
```

Comment:
is there any particular reason for this velocity (square root of vx^2 + vy^2)?

Reply:
Yes, we want to slow down the robot (i.e., scale down the velocity), if the magnitude of the request velocity for the robot is greater than the threshold. Since this method receives the x and y components of the overall velocity vector, we need to calculate the magnitude, which is done using Pythagorean's theorem.